portfolio.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE HTML>
+
+<html>
+
+<head>
+	<link rel="stylesheet" type="text/css" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
+	<link rel="stylesheet" type="text/css" href="portfolio.css">
+	<title>Amyas Gilbert's Portfolio</title>
+	<!-- <meta charset="UTF-8"> -->
```

Comment:
this line, since it is commented out, should be removed (generally we should not have commented out code in the repo, only while we are experimenting)